src/plugins/syscalls/scproto.h

```diff
@@ -105,414 +105,2979 @@
 /*
  * From http://laredo-13.mit.edu/~brendan/scproto.txt
  *
- * TODO Integrate argument extraction into syscall monitoring
  */
 
-#ifndef SYSCALLS_PROTOTYPES_H
-#define SYSCALLS_PROTOTYPES_H
+typedef struct
+{
+  const char* name;
+  const char* dir;
+  const char* dir_opt;
+  const char* type;
+} ARG;
```

Comment:
using capital names for structure definitions is not advised (usually reserved for macros). I recommend the name `win_arg_t`